server/schema/customer/res.js

```diff
@@ -59,6 +59,54 @@ const resolvers = {
         )
       })
     },
+    async loginFB(_, {input}) {
+      const user = await Customer.findOne({where: {username: input.username}})
```

Comment:
refactor this part into a reusable async function
ex:      async generateLoginJwt(username)  -> return jwt.sign(....)